optimum/exporters/tasks.py

```diff
@@ -260,6 +280,9 @@ class TasksManager:
         "vision2seq-lm": "image-to-text",
         "zero-shot-classification": "text-classification",
         "image-feature-extraction": "feature-extraction",
+        # for backward compatibility
+        "stable-diffusion": "text-to-image",
+        "stable-diffusion-xl": "text-to-image",
```

Comment:
should this be `image-to-image` for `stable-diffusion-xl` ? I don't think it matters